ios/RNInstabug/InstabugReactBridge.m

```diff
@@ -283,7 +283,15 @@ - (dispatch_queue_t)methodQueue {
     }
 }
 
-RCT_EXPORT_METHOD(networkLog:(NSDictionary *) networkData) {
+RCT_EXPORT_METHOD(networkLog:(NSDictionary *) networkData isW3cCaughted:(nonnull NSNumber  *)isW3cCaughted partialID:(nonnull NSNumber  *) partialID timestamp:(nonnull NSNumber  *) timestamp generatedW3CTraceparent:(nullable NSString  *) generatedW3CTraceparent caughtedW3CTraceparent:(nullable NSString  *) caughtedW3CTraceparent){
```

Comment:
I think the parameters are getting really long here so we can move each parameter to its own line.
```suggestion
RCT_EXPORT_METHOD(networkLog:(NSDictionary *)networkData
               isW3cCaughted:(nonnull NSNumber  *)isW3cCaughted
                   partialID:(nonnull NSNumber  *)partialID
                   timestamp:(nonnull NSNumber  *)timestamp
     generatedW3CTraceparent:(nullable NSString  *)generatedW3CTraceparent
      caughtedW3CTraceparent:(nullable NSString  *)caughtedW3CTraceparent) {
```